UI.PageModels/Pages/Admin/Booking/VideoAccessPointsPage.cs

```diff
@@ -5,31 +5,24 @@ namespace UI.PageModels.Pages.Admin.Booking;
 
 public class VideoAccessPointsPage : VhAdminWebPage
 {
-    private readonly By _addAnotherBtn = By.Id("addEndpoint");
+    private readonly By _displayNameTextField = By.Id("displayName");
+    private readonly By _defenceAdvocateSelector = By.Id("representative");
     private readonly By _nextButton = By.Id("nextButton");
 
     public VideoAccessPointsPage(IWebDriver driver, int defaultWaitTime) : base(driver, defaultWaitTime)
     {
-        WaitForElementToBeVisible(_addAnotherBtn);
+        WaitForElementToBeVisible(_displayNameTextField);
     }
-    
-    private static By RemoveDisplayName(int number) => By.Id($"removeDisplayName{number}");
 
     /// <summary>
     /// Add a list of video access points to a hearing
     /// </summary>
     /// <param name="videoAccessPoints"></param>
     public void AddVideoAccessPoints(List<VideoAccessPointsDto> videoAccessPoints)
     {
-        for (var i = 0; i < videoAccessPoints.Count; i++)
+        foreach (var vap in videoAccessPoints)
         {
-            var vap = videoAccessPoints[i];
-            
-            AddVideoEndpoint(vap.DisplayName,vap.DefenceAdvocateDisplayName,i);
-            if (i < videoAccessPoints.Count - 1)
```

Comment:
Won't removing this break the code that passes in multiple video access points?

Reply:
swapped as a foreach loop since we don't need the index position anymore. it's one form rather than a dynamic list of forms